### DISCUSSION TITLE
重新安装lantern后我的账户就变成免费版了，请客服人员帮我找回我的激活码，感谢

### DISCUSSION THREAD

**Discussion by Jiaxzh** [getlantern/lantern-forum-cn#403] [Q&A]
如题，重新安装后，我的账户就变成免费版了，请客服人员帮我找回我的激活码。
我的注册邮箱是ljhnhz@sohu.com，下面是我购买一年版VIP专业版的订单记录
![微信图片_20240406165704](https://github.com/getlantern/lantern-forum-cn/assets/31700988/2d3d169b-05a1-4d70-9b37-a32024f8f05e)

**Comment by MaxMKMax** [comment]
@Jiaxzh 您好! 可以的我让客服帮您恢复状态. 他们会手动发激活码给您. 请您过几天留有邮箱或GitHub. 谢谢!
